components/producers/producer.go

```diff
@@ -123,6 +123,7 @@ func WriteDraconOut(
 
 	stat, err := os.Stat(OutFile)
 	if Append && err == nil && stat.Size() > 0 {
+		slog.Info("appending", "issues", len(cleanIssues), "tool", toolName, "to", OutFile)
```

Comment:
You can leverage log fields for this and the tuples key:val would be more easy to understand imo

```
slog.Info(
    "appending", 
    slog.String("issues", len(cleanIssues)), 
    slog.String("tool", toolName), 
    slog.String("to", OutFile),
)
```